python/cuml/common/sparsefuncs.py

```diff
@@ -0,0 +1,99 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import math
+
+from cuml.common import with_cupy_rmm
+from cuml.common.kernel_utils import cuda_kernel_factory
+
+
+def _map_l1_norm_kernel(dtype):
```

Comment:
It would be awesome to add unit tests for each "prim" present in this file, would that be possible?

Reply:
Of course! I forgot to add tests for those, you're right.